save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsView.kt

```diff
@@ -363,4 +379,20 @@ abstract class UserSettingsView : AbstractView<UserSettingsProps, UserSettingsVi
         loadingHandler = ::classLoadingHandler,
     )
         .unsafeMap { it.decodeFromJsonString<List<OrganizationWithUsers>>() }
+
+    @Suppress("TYPE_ALIAS")
+    private suspend fun getDeletedOrganizationDtos() = get(
+        "$apiUrl/organizations/by-user?status=${OrganizationStatus.DELETED}",
```

Comment:
can we request all organization once and then filter them by their status?

Reply:
Done